picard/ui/options/metadata.py

```diff
@@ -261,4 +262,118 @@ def save_changes(self):
         self.accept()
 
 
+class ScriptExceptionSelector(PicardDialog):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.ui = Ui_ExceptionScriptSelector()
+        self.ui.setupUi(self)
+        self.ui.button_box.accepted.connect(self.save_changes)
+        self.ui.button_box.rejected.connect(self.reject)
+        self.move_view = MoveableListView(self.ui.selected_scripts, self.ui.move_up, self.ui.move_down)
+        self.ui.add_script.clicked.connect(self.add_script)
+        self.ui.remove_script.clicked.connect(self.remove_script)
+        self.ui.selected_scripts.currentRowChanged.connect(self.selected_script_changed)
+        self.ui.weighting_selector.valueChanged.connect(self.weighting_changed)
+
+        weighting_tooltip_text = (
```

Comment:
This text should be translated.

```suggestion
        weighting_tooltip_text = _(
```